packages/vue-virtual-scroller/src/components/RecycleScroller.vue

```diff
@@ -231,19 +243,31 @@ export default {
       return view
     },
 
-    unuseView (view, fake = false) {
-      const unusedViews = this.$_unusedViews
-      const type = view.nr.type
-      let unusedPool = unusedViews.get(type)
-      if (!unusedPool) {
-        unusedPool = []
-        unusedViews.set(type, unusedPool)
+    getRecycledView (type) {
+      const recycledPool = this.getRecycledPool(type)
+      if (recycledPool && recycledPool.length) {
+        const view = recycledPool.pop()
```

Comment:
I don't understand why the last item of the array needs to be deleted here

Reply:
I mean, it's literally popping the last recycled item to turn it into a non-recycled state?

Reply:
Do you want to make the last item non recyclable? Pop () will delete the last item of the array. 👉[Document details](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/pop)

Reply:
I mean, a used view is not in the recycle pool. In plain words, a view is either in the trash can (recycle pool) or on the screen, and never both at once.
This pop is paired with a push. https://github.com/Akryum/vue-virtual-scroller/blob/e07516aa5b55a63f87a326695c7448ba1e13678f/packages/vue-virtual-scroller/src/components/RecycleScroller.vue#L260

I very much know what the Array functions do. The way you're assuming otherwise is offending me.

Reply:
I'm very sorry, I understand. Thank you very much for your answer.